tests/Entity/EntityBase.php

```diff
@@ -66,7 +63,7 @@ protected function validateNotBlanks(array $fields): void
                 array_key_exists($key, $errors),
                 "{$key} key not found in errors: " . var_export(array_keys($errors), true)
             );
-            $this->assertSame('NotBlank', $errors[$key]);
+            $this->assertSame('This value should not be blank.', $errors[$key]);
```

Comment:
I don't love going to a full translation of the error instead of the key, but I accept this.